client/modules/food/components/packing/Packages.js

```diff
@@ -8,27 +8,41 @@ import {BootstrapTable, TableHeaderColumn, SizePerPageDropDown} from 'react-boot
 import 'react-bootstrap-table/dist/react-bootstrap-table.min.css'
 
 import selectors from '../../../../store/selectors'
-import {listPackages, unpackPackage} from '../../reducers/packing'
+import {listPackages, unpackPackage, receivedPackage} from '../../reducers/packing'
 
 import {Box, BoxBody, BoxHeader} from '../../../../components/box'
 import moment from 'moment'
 
 const mapStateToProps = state => ({
   loading: selectors.food.packing.loading(state),
   error: selectors.food.packing.loadError(state),
-  packages: selectors.food.packing.packages(state)
+  packages: selectors.food.packing.packages(state),
+  
 })
 
 const mapDispatchToProps = dispatch => ({
   load: () => dispatch(listPackages()),
-  unpack: packageId => dispatch(unpackPackage(packageId))
+  unpack: packageId => dispatch(unpackPackage(packageId)),
+  received: singlePackage => dispatch(receivedPackage(singlePackage)),
 })
 
 class Packages extends Component {
+  constructor() {
```

Comment:
If you make the changes I commented in the reducer you shouldn't need to keep any state or reload the packages